mappings/net/minecraft/world/dimension/Dimension.mapping

```diff
@@ -1,20 +1,45 @@
 CLASS bth net/minecraft/world/dimension/Dimension
+	FIELD a moonPhaseToSize [F
```

Comment:
Is this a static final field, or not static?

Reply:
It's `static final`.